docs/index.md

```diff
@@ -148,3 +148,10 @@ Any `.py` file stored in the `plugins` folder will be automatically loaded. All
 You can define custom grading classes in your plugin. To learn how this works, we recommend copying the code from `stringgrader.py`, renaming the class, and building a simple plugin based on `StringGrader`.
 
 We are happy to include user-contributed plugins in the repository for this library. If you have built a plugin that you would like to see combined into this library, please contact the authors through [github](https://github.com/mitodl/mitx-grading-library). We are also willing to consider incorporating good plugins into the library itself.
+
```

Comment:
Just noticed this line above (unedited in this pr):

`You can define custom grading classes in your plugin. To learn how this works, we recommend copying the code from `stringgrader.py`, renaming the class, and building a simple plugin based on `StringGrader`.`

`StringGrader` started (2 years ago?) as a toy example, I guess it's still simple enough to be a good entry point, though.

Reply:
Indeed, I remembered that we had that there... It's still a much simpler grader than literally anything else that we have, so I'm happy to keep that comment there.